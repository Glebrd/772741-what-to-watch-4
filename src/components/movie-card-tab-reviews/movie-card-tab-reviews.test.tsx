import * as React from "react";
import * as renderer from "react-test-renderer";
import MovieCardTabReviews from "./movie-card-tab-reviews";

const comments = [
  {
    "id": 1,
    "user": {
      "id": 5,
      "name": `Kate Mur`
    },
    "rating": 8.9,
    "comment": `Two detectives, a rookie and a veteran, hunt a serial killer who uses`,
    "date": `2019-05-08T14:13:56.569Z`
  }
];

it(`Movie card tab reviews renders correctly`, () => {
  const tree = renderer
    .create(
        <MovieCardTabReviews
          comments={comments}
        />).toJSON();

  expect(tree).toMatchSnapshot();
});
