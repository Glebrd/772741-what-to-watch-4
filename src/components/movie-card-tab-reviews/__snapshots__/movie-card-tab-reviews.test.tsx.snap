// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie card tab reviews renders correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Two detectives, a rookie and a veteran, hunt a serial killer who uses
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Kate Mur
          </cite>
          <time
            className="review__date"
            dateTime="2019-05-08"
          >
            May 08, 2019
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        8.9
      </div>
    </div>
  </div>
  <div
    className="movie-card__reviews-col"
  />
</div>
`;
